test/WebSites/ModelBindingWebSite/Controllers/TryUpdateModelController.cs

```diff
@@ -100,12 +100,30 @@ public class TryUpdateModelController : Controller
 
         public async Task<Employee> GetEmployeeAsync_BindToBaseDeclaredType()
         {
-            var employee = new Employee();
-            await TryUpdateModelAsync<Person>(
+            Person employee = new Employee();
+            await TryUpdateModelAsync(
                 employee,
-                prefix: string.Empty);
+                employee.GetType(),
+                string.Empty,
+                (content, propertyName) => true);
+
+            return (Employee)employee;
```

Comment:
`employee` is of type `Person` but action returns `Employee`. Hence I am casting